src/commands/dev/lib/dev-builder.ts

```diff
@@ -165,9 +165,9 @@ export async function executeBuild(
     devServer.output.debug(`Creating build process for ${entrypoint}`);
     buildProcess = await createBuildProcess(
       match,
-      devServer.buildEnv,
+      buildEnv,
```

Comment:
Should `env` be passed too?

Reply:
No, this is for the build process. We don't want to make the runtime env available there, except through `meta`.

Reply:
No